drivers/storage/portworx/testspec/portworxSCC.yaml

```diff
@@ -17,7 +17,6 @@ kind: SecurityContextConstraints
 metadata:
   name: portworx
   resourceVersion: "1"
-priority: 2
```

Comment:
the `-` is missing

Reply:
what do you mean? - means this line is removed

Reply:
Ohh nvm I thought it's a yaml list...